.format_text_tasks/.form

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Формы</title>
+</head>
+<body>
+    <table border="1" cellspacing="1" cellpadding="28">
+        <tr>
+            <td colspan="3">
+                <p>Ваше имя:</p>
+                <form>
+                    <input type="text">
+                </form>
+                <p>Ваше фото:</p>
+                <form>
+                    <input type="file">
+                </form>
+                <p>Введите пароль:</p>
+                <form>
+                    <input type="password">
+                </form>
+                <em>Это поле для тех, у кого особый пароль:</em>
+                <form>
+                    <input type="password">
+                </form>
```

Comment:
ypu should wrap the table in form. Not create form element for every input. Form it is number of inputs